src/protocol/message/mod.rs

```diff
@@ -89,82 +98,89 @@ pub enum Message {
     FilterClear,
 }
 
+macro_rules! enode_with_header_prefix {
```

Comment:
Nice! 👍

Reply:
Almost missed it but:
```suggestion
macro_rules! encode_with_header_prefix {
```